apps/aot/src/app/events/[year]/leaderboard/[day]/page.tsx

```diff
@@ -1,7 +1,5 @@
-import DailyLeaderboard from '../_components/daily-leaderboard';
-
 // export const revalidate = 10; Need next.js v15
 
 export default function Page({ params: { year, day } }: { params: { year: string; day: string } }) {
-  return <DailyLeaderboard adventYear={year} adventDay={day} />;
+  return <></>;
```

Comment:
if theres nothing in here I think we can delete it or does it break?

my nextjs fu is bad

Reply:
nah the leaderboard is only implemented for 2023 rn, lemme add it to this page too.